.circleci/config.yml

```diff
@@ -101,7 +102,7 @@ jobs:
           command: |
             pwd
             ls
-            python -m pytest -vv litellm/tests/ -x --junitxml=test-results/junit.xml --durations=5 -k "not test_python_38.py"
+            python -m pytest -vv litellm/tests/ -x --junitxml=test-results/junit.xml --durations=5 -k "not test_python_38.py" -n auto
```

Comment:
<img width="713" alt="image" src="https://github.com/BerriAI/litellm/assets/8990777/98e8cddb-1afc-45c7-b998-14a3df19844f">

This wasn't used it seems

Reply:
seems to run here @jamesbraza https://app.circleci.com/pipelines/github/BerriAI/litellm/11589/workflows/01da8c12-1dba-4176-a233-dfae214ac2d4/jobs/27082

it's pretty hard to debug the errors though :(

Reply:
Wow 36 workers, your CI instance is huge haha

Is your CI flaky as is? Usually GitHub Actions will auto-scroll to the failure point for me